geopandas/plotting.py

```diff
@@ -102,9 +106,8 @@ def plot_series(s, colormap='Set1', axes=None, **color_kwds):
     """
     import matplotlib.pyplot as plt
     if axes is None:
-        fig = plt.gcf()
-        fig.add_subplot(111, aspect='equal')
-        ax = plt.gca()
+        fig, ax = plt.subplots(figsize=figsize)
```

Comment:
This is actually a change in behaviour, as before it was using an open figure if it existed, and if not creating a new one. 
(But not sure if the old behaviour was better ..)